frontend/src/components/Pages/FrontPage/MissionOverview/UpcomingMissionView.tsx

```diff
@@ -93,19 +92,17 @@ export function UpcomingMissionView() {
         })
     }, [])
 
-    useEffect(() => {
-        const installationCode = sessionStorage.getItem('assetString')
-        if (installationCode !== assetString) {
-            setAssetString(installationCode as string)
-        }
-    }, [sessionStorage.getItem('assetString')])
-
     useEffect(() => {
         const id = setInterval(() => {
-            apiCaller.getEchoMissions(assetString).then((missions) => {
-                const mappedEchoMissions: Map<string, EchoMission> = mapEchoMissionToString(missions)
-                setEchoMissions(mappedEchoMissions)
-            })
+            const installationCode = sessionStorage.getItem('assetString')
+            if (!installationCode || installationCode == '') {
+                setEchoMissions(mapEchoMissionToString([]))
```

Comment:
Is this possible to do?
```suggestion
                setEchoMissions([])
```

Reply:
this does not work, but changing to new Map<String, EchoMission> instead